### PR TITLE
fix: add type in publishPaymentNotification

### PR DESCRIPTION
**Pull request by im-adithya**
## ⚠️ Requires Input

This change starts calling ln.keysend.success for keysend payments (as desired), but that would mean we are not calling the success/failure notifications here: https://github.com/getAlby/lightning-browser-extension/blob/master/src/extension/background-script/events/index.js#L13-L14
Wanted to know if that's also done on purpose or I should add notifs for keysend payments

### Describe the changes you have made in this PR

The `ln.keysend.success` event isn't being triggered at all because we only publish the `ln.sendPayment.success` event everytime.

### Link this PR to an issue [optional]

Fixes _#ISSUE-NUMBER_

### Type of change

(Remove other not matching type)

- `fix`: Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

Didn't test yet, require input.

### Checklist

- [x] My code follows the style guidelines of this project and performed a self-review of my own code
- [x] New and existing tests pass locally with my changes
- [x] I checked if I need to make corresponding changes to the documentation (and made those changes if needed)

### CODE REVIEW

**Comment by bumi** [comment]
it makes sense to me, but we should add the according subscriptions also to the keysend payments for the success/error notifications

File: src/extension/background-script/events/index.js [hunk 1]
```diff
@@ -2,16 +2,18 @@ import PubSub from "pubsub-js";
...
+  lnurlAuthSuccessNotification,
+  paymentFailedNotification,
+  paymentSuccessNotification,
 } from "./notifications";
 import { persistSuccessfullPayment } from "./persistPayments";
 
 const subscribe = () => {
   PubSub.subscribe("ln.sendPayment.success", paymentSuccessNotification);
   PubSub.subscribe("ln.sendPayment.failed", paymentFailedNotification);
+  PubSub.subscribe("ln.keysend.success", paymentSuccessNotification);
```
Comment (rolznz) [inline src/extension/background-script/events/index.js:15]: should we have a for loop on ['sendPayment', 'keysend'] here? looks like there are now 3 subscriptions which are the same for both

**Comment by rolznz** [comment]
I am just wondering now that this issue comes because of how the spec was implemented. The name "sendPayment" is a generic name for a payment (but actually just bolt11) whereas "keysend" is a named from the specific type of payment. What happens if we add other ways of payment too? what about paying using webln.lnurl()? are we going to subscribe to all these individually?

I don't have answers, just maybe things to consider. CC @bumi

**Review by rolznz — APPROVED** [review]

File: src/extension/background-script/events/index.js (hunk 1 above)
Comment (im-adithya) [reply src/extension/background-script/events/index.js]: Makes sense! Done!

**Comment by bumi** [comment]
lnurl payments are `sendPayment` calls with a bolt11. 
other forms can be discussed when there will be other forms. (bolt12 🌈🦄)

**Pull request merged**